app/dashboard/templates/ftux/avatar.html

```diff
@@ -92,7 +92,7 @@ <h3 id="section-title">
   </div>
   <!-- Button Row -->
   <div id="avatar-builder__actions">
-    <p id="prev-btn" class="button button--primary-o" onclick="changeStep(-1)" style="float: left;">{% trans "Previous" %}</p>
```

Comment:
Sorry for this, man. Missed that there will be 2 ids since we were rendering the entire FTUX out. Nice catch. Thanks.

Reply:
No worries :P